### DISCUSSION TITLE
Latest bundle installer is blocked by SentinelOne

### DISCUSSION THREAD

**Discussion by mgscano** [SafeExamBrowser/seb-win-refactoring#382] [Q&A] [answered]
As per the subject, SentinelOne behavioral AI labels the latest bundle installer "SEB_3.3.2.413_SetupBundle.exe" as malware, and removes it at once.
SHA1 fingerprint of blocked installer is 386eae441d116f4f6047c3fcc30c6f4a8c61fcde

Could you please confirm the original file hash, so I can report it as safe?

All the Best,
Marco

**Comment by dbuechel** [comment, marked as answer]
That is incorrect and might be the cause of the issue? The correct SHA1 file hash of the original setup bundle for version 3.3.2.413 is `5b1afe353aefe48942822f89d9f1e2b1f9833750`, see https://safeexambrowser.org/download_en.html.

**Reply by mgscano** [reply]
Thanks for that, will check on the original exe then.
Anyway, this installer behaves in such a weird way, among all the vc redist packages, it downloads a 1KB file from 93.184.220.29, that is pretty suspicious, and really behaves in a way that is too similar to your next emotet ...
Gonna mark it as safe for my users, but a more straightforward installer would eventually bring less false positives.

**Reply by dbuechel** [reply]
Our installer and all binaries are correctly signed, could you please verify the signature on that setup you have with the wrong checksum?

The setup bundle should only download and install the three runtime dependencies (as e.g. stated in the README or release info here on GitHub), this indeed sounds suspicious. Where did you get that installer from?
